src/165490.c

```diff
@@ -34,15 +80,92 @@ void filemenu_draw_contents_mono(
     }
 }
 
-INCLUDE_ASM(s32, "165490", filemenu_draw_contents_option_left);
+void filemenu_draw_contents_option_left(
+    MenuPanel* menu,
+    s32 baseX, s32 baseY,
+    s32 width, s32 height,
+    s32 opacity, s32 darkening
+) {
+    if (menu->page != 2) {
+        if ((menu->unk_00.s & 0x00FFFF00) == 0x200) {
+            filemenu_set_cursor_goal_pos(53, baseX, baseY + 8);
+        }
+        filemenu_draw_message(filemenu_get_menu_message(9), baseX + 8, baseY + 2, 255, 0, 1);
+    }
+}
+
+void filemenu_draw_contents_option_center(
+    MenuPanel* menu,
+    s32 baseX, s32 baseY,
+    s32 width, s32 height,
+    s32 opacity, s32 darkening
+) {
+    s32 msgIdx;
+    s32 xOffset;
+    s32 yOffset;
+
+    switch (menu->page) {
+        case 1:
+        case 2:
+        case 3:
+        case 4:
+            msgIdx = 10;
+            xOffset = 18;
+            yOffset = 0;
+            if ((menu->unk_00.s & 0xFFFF00) == 0x10200) {
```

Comment:
Like on line 139 can you make both these literals the same width?
```suggestion
            if ((menu->unk_00.s & 0xFFFF00) == 0x010200) {
```